dkg/sync/server.go

```diff
@@ -109,11 +110,12 @@ func (s *Server) isConnected(pID peer.ID) bool {
 }
 
 // setConnected sets the shared connected state for the peer.
-func (s *Server) setConnected(pID peer.ID) {
+func (s *Server) setConnected(ctx context.Context, pID peer.ID) {
```

Comment:
another way is to return the length and log with context on the outside

Reply:
Done!